ui/app/pages/send/send-content/send-row-wrapper/send-row-wrapper.component.js

```diff
@@ -34,7 +34,39 @@ export default class SendRowWrapper extends Component {
       <div className="send-v2__form-row">
         <div className="send-v2__form-label">
           {label}
-          {showError && <SendRowErrorMessage errorType={errorType}/>}
+          {customLabelContent}
+        </div>
+        <div className="send-v2__form-field-container">
+          <div className="send-v2__form-field">
+            {formField}
+          </div>
+          <div>
+            {showError && <SendRowErrorMessage errorType={errorType} />}
+            {!showError && showWarning && <SendRowWarningMessage warningType={warningType} />}
+          </div>
+        </div>
+      </div>
+    )
+  }
+
+  renderFormRow () {
+    const {
+      children,
+      errorType = '',
+      label,
+      showError = false,
+      showWarning = false,
+      warningType = '',
+    } = this.props
+
+    const formField = Array.isArray(children) ? children[1] || children[0] : children
+    const customLabelContent = children.length > 1 ? children[0] : null
```

Comment:
If we're uncertain that children is an array, we should include that in the condition here:

```suggestion
    const customLabelContent = (Array.isArray(children) && children.length) > 1 ? children[0] : null
```

Reply:
In which case we might as well introduce an if statement to handle the combined cases

Reply:
This one I didn't change, it must have just moved it, but thank you for the comment!